src/index.js

```diff
@@ -1,4 +1,152 @@
+	const city = document.createElement('h2');
+	city.innerHTML = data;
+	city.classList.add('city-name');
+	$('.town-name').appendChild(city);
+	timezone(time);
+	const img = weatherimage(image);
+	img.classList.add('image');
+	$('.town-name').appendChild(img);
+};
+
+const degrees = (k, image) => {
+	const c = convertToCelsius(k);
+	const temp = document.createElement('h3');
+	temp.innerHTML = `${c} &#8451`;
+	$('.degrees').appendChild(temp);
+	const img = weatherimage(image);
+	img.classList.add('image');
+	$('.degrees').appendChild(img);
+};
+
+
+const latitute = (long, lati) => {
+	const lon = document.createElement('h3');
+	lon.innerHTML = `Lon: ${long}&#176`;
+	const lat = document.createElement('h3');
+	lat.innerHTML = `lat: ${lati}&#176`;
+	$('.sunset').appendChild(lat);
+	$('.sunset').appendChild(lon);
+};
+
+const sys = (data, image) => {
+	const img = weatherimage(image);
+	img.classList.add('image');
+	$('.sys').appendChild(img);
+	// eslint-disable-next-line no-restricted-syntax
+	for (const [key, value] of Object.entries(data)) {
+		const p = document.createElement('p');
+		p.innerHTML = `${key}:  ${value}`;
+		$('.sys').appendChild(p);
+	}
+};
+
+const mainData = (data, image) => {
+	const img = weatherimage(image);
+	img.classList.add('image');
+	$('.main-data').appendChild(img);
+	// eslint-disable-next-line no-restricted-syntax
+	for (const [key, value] of Object.entries(data)) {
+		const p = document.createElement('p');
+		p.innerHTML = `${key}:  ${value}`;
+		$('.main-data').appendChild(p);
+	}
+};
+
+const weather = (data, image) => {
+	const img = weatherimage(image);
+	img.classList.add('image');
+	$('.weather').appendChild(img);
+	const heading = document.createElement('p');
+	heading.innerHTML = 'Weather';
+	$('.weather').appendChild(heading);
+	// eslint-disable-next-line no-restricted-syntax
+	for (const [key, value] of Object.entries(data)) {
+		const p = document.createElement('p');
+		p.innerHTML = `${key}:  ${value}`;
+		$('.weather').appendChild(p);
+	}
+};
+
+const clearNodes = () => {
+	$('.town-name').innerHTML = '';
+	$('.degrees').innerHTML = '';
+	$('.sunset').innerHTML = '';
+	$('.sys').innerHTML = '';
+	$('.main-data').innerHTML = '';
+	$('.weather').innerHTML = '';
+};
+
+const changeBg = () => {
+	const weatherData = JSON.parse(localStorage.getItem('city'));
+	const sky = weatherData.weather[0].main;
+	if (sky === 'Clear') {
+		$('body').style.backgroundImage = 'url(../dist/img/weather.jpg)';
+	} else if (sky === 'Snow') {
+		$('body').style.backgroundImage = 'url(../dist/img/snow.gif)';
+	} else if (sky === 'Rain') {
+		$('body').style.backgroundImage = 'url(../dist/img/rain.gif)';
+	} else if (sky === 'Clouds') {
+		$('body').style.backgroundImage = 'url(../dist/img/cloudy.gif)';
+	} else {
+		$('body').style.backgroundImage = 'url(../dist/img/weather.jpg)';
+	}
+};
+
+const getApi = () => {
+	const city = $('#city-name').value;
+	const country = $('#country').value;
```

Comment:
I don't see the effect of the country. no matter the country i pick, the city result is returned. example country set to `spain` and city  `abuja` still return the data for abuja.
- If you like take out the country completely.